docs/installation/centos.md

```diff
@@ -12,153 +12,185 @@ parent = "smn_linux"
-For CentOS-6.5, the Docker package is part of [Extra Packages
-for Enterprise Linux (EPEL)](https://fedoraproject.org/wiki/EPEL) repository,
-a community effort to create and maintain additional packages for the RHEL distribution.
+    This ensures your user is running with the correct permissions.
 
-Firstly, you need to ensure you have the EPEL repository enabled. Please
-follow the [EPEL installation instructions](
-https://fedoraproject.org/wiki/EPEL#How_can_I_use_these_extra_packages.3F).
+4. Verify your work by running `docker` without `sudo`.
 
-For CentOS-6, there is a package name conflict with a system tray application
-and its executable, so the Docker RPM package was called `docker-io`.
+		$ docker run hello-world
+ 
+## Start the docker daemon at boot
 
-To proceed with `docker-io` installation on CentOS-6, you may need to remove the
-`docker` package first.
+To ensure Docker starts when you boot your system, do the following:
 
-    $ sudo yum -y remove docker
+      $ sudo chkconfig docker on
 
-Next, let's install the `docker-io` package which will install Docker on our host.
-
-    $ sudo yum install docker-io
-
-Please continue with the [Starting the Docker daemon](#starting-the-docker-daemon).
-
-### Uninstallation
-
-To uninstall the Docker package:
-
-    $ sudo yum -y remove docker-io
-
-The above command will not remove images, containers, volumes, or user created
-configuration files on your host. If you wish to delete all images, containers,
-and volumes run the following command:
-
-    $ rm -rf /var/lib/docker
-
-You must delete the user created configuration files manually.
-
-## Manual installation of latest Docker release
-
-While using a package is the recommended way of installing Docker,
-the above package might not be the current release version. If you need the latest
-version, [you can install the binary directly](
-https://docs.docker.com/installation/binaries/).
-
-When installing the binary without a package, you may want
-to integrate Docker with Systemd. For this, install the two unit files
-(service and socket) from [the GitHub
-repository](https://github.com/docker/docker/tree/master/contrib/init/systemd)
-to `/etc/systemd/system`.
-
-Please continue with the [Starting the Docker daemon](#starting-the-docker-daemon).
-
-## Starting the Docker daemon
-
-Once Docker is installed, you will need to start the docker daemon.
-
-    $ sudo service docker start
-
-If we want Docker to start at boot, we should also:
-
-    $ sudo chkconfig docker on
-
-Now let's verify that Docker is working. First we'll need to get the latest
-`centos` image.
-
-    $ sudo docker pull centos
-
-Next we'll make sure that we can see the image by running:
-
-    $ sudo docker images centos
+If you need to add an HTTP Proxy, set a different directory or partition for the
+Docker runtime files, or make other customizations, read our Systemd article to
+learn how to [customize your Systemd Docker daemon options](/articles/systemd/).
```

Comment:
This may be confusing, because (afaik) CenOS/RHEL/Fedora 6 uses SysVinit; 7.x uses SystemD. The installation steps above assume installing on 6.x